--- /usr/local/lib/python3.8/dist-packages/nest/experiment/plotter/tc.py	2021-10-05 11:48:46.434954881 +0530
+++ tc.py	2021-10-05 18:24:27.455865060 +0530
@@ -5,6 +5,7 @@
 
 import logging
 import matplotlib.pyplot as plt
+import numpy as np
 
 from .common import simple_plot
 from ..pack import Pack
@@ -83,6 +84,12 @@
         filename = f"{node}_{interface}_{qdisc}_{param}.png"
         Pack.dump_plot("tc", filename, fig)
         plt.close(fig)
+        
+        filename2 = f"{node}_{interface}_{qdisc}_{param}.png"
+        np.warnings.filterwarnings('ignore', category=np.VisibleDeprecationWarning)                 
+        dat = np.array([timestamp, stats_params[param]])
+        dat_array= np.column_stack(dat)
+        Pack.dump_dat(filename2, dat_array)
 
 
 def plot_tc(parsed_data):
